src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -171,16 +171,21 @@ class BaseExpensiTextInput extends Component {
                             ]}
                         >
                             {hasLabel ? (
-                                <ExpensiTextInputLabel
-                                    label={label}
-                                    labelTranslateX={
-                                        ignoreLabelTranslateX
-                                            ? new Animated.Value(0)
-                                            : this.state.labelTranslateX
-                                    }
-                                    labelTranslateY={this.state.labelTranslateY}
-                                    labelScale={this.state.labelScale}
-                                />
+                                <>
+                                    {/* Adding this background to the label only for multiline text input,
+                                    to prevent text overlaping with label when scrolling */}
+                                    {multiline && <View style={styles.expensiTextInputLabelBackground} />}
```

Comment:
Am I correct in thinking that the only reason we don't add this view for single-line inputs is because of the effect it causes when an input is auto-filled? Did you see or investigate [this comment](https://github.com/Expensify/App/issues/5924#issuecomment-947393403)?

Reply:
Yes, it is only set to multiline to avoid seeing the white background when isn't multiline and it has autofill. And it look all the container blue(or whatever the color set depending on the browser).

I did investigate on it but there are some points to consider when trying to change the background set by the browser.

See https://github.com/Expensify/App/pull/5704#issuecomment-947258907